Formula/goreman.rb

```diff
@@ -0,0 +1,50 @@
+require "language/go"
+
+class Goreman < Formula
+  desc "Foreman clone written in Go"
+  homepage "https://github.com/mattn/goreman"
+  url "https://github.com/mattn/goreman/archive/v0.1.1.tar.gz"
+  sha256 "d234ae81a397ab04338f26e30db79811715202413e0a3b9ff20395df7df675df"
+
+  bottle do
```

Comment:
Apologies but I'm not sure how to initialize these values or even whether I can.

Reply:
The bottle block should be removed, it will be added when the formula is merged.